Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEventListener.java

```diff
@@ -175,6 +177,32 @@ public class PlayerEventListener implements Listener {
             Material.WRITABLE_BOOK,
             Material.WRITTEN_BOOK
     );
+    private final Set<String> DYES;
```

Comment:
This should be static final, and in best case you can first create the mutable set and then use Set.copyOf to store an immutable variant